core/templates/dev/head/components/common-layout-directives/common-elements/promo-bar.directive.ts

```diff
@@ -22,15 +22,13 @@ require('domain/utilities/url-interpolation.service.ts');
 require('services/promo-bar.service.ts');
```

Comment:
Please remove the line above this one. The `require('domain/utilities/url-interpolation.service.ts');`.

Reply:
Done